src/libraries/Microsoft.PowerFx.Connectors/Tabular/Services/CdpTable.cs

```diff
@@ -110,12 +110,11 @@ protected override async Task<IReadOnlyCollection<DValue<RecordValue>>> GetItems
 
             string queryParams = (odataParameters != null) ? "&" + odataParameters.ToQueryString() : string.Empty;
 
-            Uri uri = new Uri(
-                   (_uriPrefix ?? string.Empty) +
+            var uri = (_uriPrefix ?? string.Empty) +
                    (CdpTableResolver.UseV2(_uriPrefix) ? "/v2" : string.Empty) +
-                   $"/datasets/{(DatasetMetadata.IsDoubleEncoding ? DoubleEncode(DatasetName) : SingleEncode(DatasetName))}/tables/{Uri.EscapeDataString(TableName)}/items?api-version=2015-09-01" + queryParams, UriKind.Relative);
```

Comment:
>. [](http://example.com/codeflow?start=217&length=1)

why this change?

Reply:
encoded characters for query parameters which has Unicode chars like Biométricos then the encoded Table name/query params value lost on calling uri.ToString() function and hence fixing the same issue which will construct the Url as below with the above fix

items?api-version=2015-09-01&$filter=%28Name%20eq%20%27Biom%C3%A9tricos%27%29&$select=Name&$top=1000